modules/sbo-categories-of-exposable-binding-data.adoc

```diff
@@ -23,6 +23,23 @@ metadata:
 ----
 
 
+== Exposing a constant value as the binding item
+The following examples show how to expose constant value as annotations:
+
+.Example: Exposing constant value as annotations
```

Comment:
I would drop `as annotations`, and leave just `Exposing constant value`

Reply:
Addressed.

Reply:
@Srivaralakshmi this drop of `as annotations` would be applied to all example titles.